File: challenges/1-media-alunos/php/davidspader/challenge.php
<?php

$notas = $argv;
$soma = 0;
$qtdNotas = count($notas) - 1;

foreach($notas as $nota){
    if(is_numeric($nota)){
        $soma+= $nota; 
    }
}

$media = $soma / $qtdNotas;

echo "A sua média aritmética é: $media";

File: challenges/1-media-alunos/python/evertoont/challenge.py
while True:
    try:
        qtd_notas = int(input('Digite a quantidade de notas: '))

        soma = 0
        for indice in range(qtd_notas):
            notas = float(input(f'Nota {indice+1}: '))
            soma += notas

        media = soma/qtd_notas

        print(f'\nMédia: {media:.2f}')

    except ValueError:
        print('\nValor inválido!\n')

    else:
        break
